cmd/gokr-packer/packer.go

```diff
@@ -106,6 +107,14 @@ You can also create your own certificate-key-pair (e.g. by using https://github.
 	testboot = flag.Bool("testboot",
 		false,
 		"Trigger a testboot instead of switching to the new root partition directly")
+
+	deviceType = flag.String("device_type",
+		"",
+		"Device type identifier (defined in github.com/gokrazy/internal/deviceconfig)")
+
+	mbrOnlyWithoutGpt = flag.Bool("mbr_only_without_gpt",
```

Comment:
Can’t this flag be derived from the device config? I’d rather not expose such a low-level detail, and instead focus on letting the user specify the device type instead.